js/foundation.reveal.js

```diff
@@ -184,13 +184,14 @@ class Reveal {
    * @fires Reveal#open
    */
   open() {
+    // either update or replace browser history
     if (this.options.deepLink) {
       var hash = `#${this.id}`;
 
-      if (window.history.pushState) {
-        window.history.pushState(null, null, hash);
+      if (this.options.updateHistory) {
```

Comment:
_For now_ we have to support IE 9. So we should keep the `window.history.pushState` check.